bpmn/Dockerfile

```diff
@@ -26,13 +26,20 @@ RUN /usr/local/bin/pkg --targets ${NODE}-${PLATFORM}-${ARCH} . -o app.bin
 # Create the image
 FROM alpine:3.13.2
 
-COPY --from=builder /app/app.bin /node/bin/bpmn
-COPY assets /app/assets
+RUN apk add --no-cache --update chromium
+
+ENV USER_ID=1000 USER=kroki
```

Comment:
I don't think we should use environment variables, I'm fine with small "code duplication".
As far as I know, the environment variables declared with `ENV` will persist when a container is run from the resulting image.
In this case, `USER_ID` and `USER` are build-only constants.

Reply:
👍  Agree. I'll change.